src/pages/authorization/sign-in-with-livechat/index.mdx

```diff
@@ -9,7 +9,7 @@ desc: 'Some apps you build may need to access data from the LiveChat users accou
 
 # Introduction
 
-The **Sign in with LiveChat** flow is the easiest way to get access to basic information in LiveChat accounts. It allows you to quickly build an app that can access that info.
+The **Sign in with LiveChat** flow is the easiest way to get access to basic information in LiveChat and HelpDesk accounts. It allows you to quickly build an app that can access that info.
```

Comment:
@ale-kacperczyk, what are we doing with this document? At some point, we will have Product SDK, so should we for now introduce changes and add HD here or just leave it like it is, only for LiveChat?